meta-leda-distro-container/recipes-sdv/sdv-containers/kuksa-gps-feeder-container_git.bb

```diff
@@ -0,0 +1,30 @@
+# /********************************************************************************
+# * Copyright (c) 2023 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * https://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+SUMMARY = "Eclipse Kuksa - GPS Feeder container image"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+require ../images/sdv-image-container.bb
+
+IMAGE_INSTALL += " \
```

Comment:
Should this be a DEPENDS, instead of a direct image-install inside a recipe?

Reply:
No, this is an image recipe, so IMAGE_INSTALL is correct.

Reply:
Ah, okay then, mistake